Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -9860,6 +9862,120 @@ void WebPage::hasActiveNowPlayingSessionChanged(bool hasActiveNowPlayingSession)
     send(Messages::WebPageProxy::HasActiveNowPlayingSessionChanged(hasActiveNowPlayingSession));
 }
 
+void WebPage::simulateClickOverText(const String& targetText, CompletionHandler<void(bool)>&& completion)
```

Comment:
I'd call this something like `simulateClickOverFirstMatchingTextInViewport` or something

Reply:
Makes sense! I'm renaming this to `simulateClickOverFirstMatchingTextInViewportWithUserInteraction` (to also clarify the `m_userIsInteracting` bit below).